flight/Modules/Sensors/sensors.c

```diff
@@ -529,23 +529,26 @@ void update_optical_flow(struct pios_sensor_optical_flow_data *optical_flow)
  * @param [in] rangefinder raw rangefinder data
  */
 #if defined (PIOS_INCLUDE_RANGEFINDER)
-static void update_rangefinder(struct pios_sensor_rangefinder_data *rangefinder)
+static void update_rangefinder(struct pios_sensor_rangefinder_data *data)
 {
-	RangefinderDistanceData rangefinderAltitude;
-	RangefinderDistanceGet(&rangefinderAltitude);
+	RangefinderData rangefinder = { 0 };
 
-	AttitudeActualData attitude;
-	AttitudeActualGet(&attitude);
+	rangefinder.Range = data->range;
+	rangefinder.Velocity = data->velocity;
 
-	rangefinderAltitude.Range = rangefinder->range;
+	if (!data->range_status) {
```

Comment:
OK, force push'd.  I never know how much to change (old code was if (range_status == 0).